### ISSUE TITLE
Unable to accept the quest In a Pickle

### ISSUE THREAD

**Issue by zuzupetals** [EdenServer/community#1661]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

**Comment by zuzupetals** [comment]
The npc Chamama only says "Welcome to the Rarab Tail Hostelry, dearie etc." and "Windurst is a fun place best enjoyed etc." She will not give me the quest. I have tried zoning and talking to her multiple times.

**Comment by Crissski** [comment]
Did you complete at least 1 other quest? This requires Windurst reputation of 1. (I think you only have to complete 1 quest prior)

**Comment by RottenUnagi** [comment]
I've also experienced this from time to time on many multiple chars, both on Nasomi and Eden. I've never figured out why some chars can get it right away, and some can't. I don't think completing some other quest first is necessarily the solution? The Justice Badge quest is *always* the first quest I do on Windurst characters, as well as the Fish Scale Shield quest, yet sometimes I'm still unable to flag the 'Pickle' quest until later at random *shrug*. Besides, these other quests I've mentioned also say "Fame 1" on wiki, as well as many others, but there's no problem flagging them without having completed any other quests.

@zuzupetals I'd recommend just carrying on for now, do other quests etc. meanwhile, and try again later. You should likely be able to flag the quest at some point, though who knows what the actual trigger is to unlock it for you.

**Comment by Althaia** [comment]
I encountered difficulty flagging this quest as well, but only the first time; after we got our repartee going, she has been quite accommodating ever since for repeating the quest.

And it was definitely not the first quest I tried.

What worked for me was talking to Chamama many, many times. Just when I was about to give up and call the quest bugged, she threw me a bone and triggered the quest for me. ;P
